lnrpc/rpc.proto

```diff
@@ -2295,6 +2295,25 @@ message Payment {
 
     ///  The fee paid for this payment in milli-satoshis
     int64 fee_msat = 12 [json_name = "fee_msat"];
+
+    /// The HTLCs made in attempt to settle the payment [EXPERIMENTAL].
+    repeated HTLCAttempt htlcs = 13[json_name = "htlcs"];
+}
+
+message HTLCAttempt {
+        enum HTLCStatus {
+                IN_FLIGHT = 0;
+                SUCCEEDED = 1;
+                FAILED = 2;
+        }
+
+        HTLCStatus status = 1 [json_name = "status"];
+
+        repeated string path = 2 [json_name = "path"];
+
+        int64 attempt_time = 3 [json_name = "attempt_time"];
+
+        int64 resolve_time = 4 [json_name = "resolve_time"];
```

Comment:
Same comment as before, for review I'd prefer having the code using this proto in this commit too.